packages/web-twig/src/Resources/components/ScrollView/ScrollView.stories.twig

```diff
@@ -2,19 +2,29 @@
 
 {% block content %}
 
-    {# ScrollView Default #}
-    {% include '@components/ScrollView/stories/ScrollViewDefault.twig' %}
+    <DocsSection title="Vertical Scrolling">
+        {% include '@components/ScrollView/stories/ScrollViewDefault.twig' %}
+    </DocsSection>
 
-    {# ScrollView Horizontal #}
-    {% include '@components/ScrollView/stories/ScrollViewHorizontal.twig' %}
+    <DocsSection title="Horizontal Scrolling">
+        {% include '@components/ScrollView/stories/ScrollViewHorizontal.twig' %}
+    </DocsSection>
 
-    {# ScrollView Horizontal with Container Breakout #}
-    {% include '@components/ScrollView/stories/ScrollViewHorizontalBreakout.twig' %}
+    <DocsSection title="Horizontal Scrolling with Container Breakout">
+        {# Use Grid to prepare the ground for container breakout (inside of the included example). #}
+        <Grid>
+            <GridSpan over={12}>
```

Comment:
🤔

Reply:
maybe we could add a modificator to the docssection to not use grid at all so we don't need this?